app/features-json/project_json_controller.rb

```diff
@@ -25,5 +32,80 @@ class ProjectJSONController < AuthenticatedControllerBase
 
       return project_view_model.to_json
     end
+
+    post HOME do
+      REQUIRED_PARAMS = Set["lane", "repo_org", "repo_name", "project_name", "trigger_type"]
```

Comment:
this should be lower case

Reply:
Am i doing constants wrong?

Reply:
Yeah, as far as I'm aware, it's not really idiomatic to Ruby to use constants like that. It can get confusing for maintainers because constants are typically defined on a class, so when somebody comes in to read this code and fix a bug, people might see `REQUIRED_PARAMS` in one place, and assume this is defined for the entire class.

Reply:
done